wasm/demo/src/main.ts

```diff
@@ -23,7 +23,7 @@ class ExampleDIDResolver implements DIDResolver {
   }
 
   async resolve(did: String): Promise<DIDDoc | null> {
```

Comment:
BTW in core didcomm rust Example resolvers are part of core code (IIRC they are exported as one of public interfaces), but in wasm Example resolvers are available in Demo only, and not for end users. Is it intended?

Reply:
It would be nice to have, but there is no good way to add javascript snippets. So, just keep in demo, tests and documentation for now.